mkdocs.yml

```diff
@@ -0,0 +1,61 @@
+# Project information
+site_name: 'zec-qt-wallet Docs'
+site_description: 'zec-qt-wallet is a z-Addr first, Sapling compatible wallet and full node for zcashd that runs on Linux, Windows and macOS'
+site_author: 'Zcash Foundation'
+site_url: 'https://github.com/ZcashFoundation/zec-qt-wallet'
```

Comment:
This will be needed to be updated once a custom domain is set as this should be the location of the docs